_posts/blog/2018-11-07/2018-11-07-big-change-hard-work-and-strong-growth-openbazaar-2-turns-1.markdown

```diff
@@ -0,0 +1,53 @@
+
+![OpenBazaar 2 Turns 1](OpenBazaar_2_Turns_1.jpg "OpenBazaar 2 Turns 1")
+
+> Over 100,000 users have experienced the power of **OpenBazaar 2.0** since it launched and it keeps growing bigger and stronger. All of our hard work continues to drive us collectively toward a better future for crypto users.
+
+#### The Road to Here
+
+We knew we were onto something great with version 1.0 of OpenBazaar. We used exciting, emerging technologies to build it but we discovered limits that we wanted to keep pushing to get the experience right for our users.
+
+Development of OpenBazaar 2.0 started many months before its release late last year and took many hours of planning and designing from a large group of team members covering visual design, technical architecture, UX and UI development, community evangelism and marketing strategy.  Some critical decisions were made to improve the experience of OpenBazaar, like **scrapping our fully decentralized search feature** in favor of using **third party search engines** (much like how Bittorrent operates), and replacing our shoddy p2p protocol with Protocol Labs’ [IPFS](https://ipfs.io/) and [libp2p](https://libp2p.io/). This made our network connectivity improve greatly (though we still have a lot of work to do) and laid the groundwork for getting us to **mobile** and **web** applications.
+
+#### Bitcoin Breakdown
+
+These changes took a long time to implement and deliver, but the team did so fairly seamlessly [on November 1st, 2017](https://openbazaar.org/blog/openbazaar-20-is-live/). There wasn’t much time to celebrate though as a transaction fee crisis was happening in parallel with a massive uptick in interest in Bitcoin. This sent the team into disarray as we struggled to find users who wanted to pay $30 in fees just to buy something like a $20 t-shirt in the marketplace. 
+
+After some consternation we decided the only solution was to expand our payment options to include other currencies to ease the scalability pains felt by our users and so in [January](https://openbazaar.org/blog/openbazaar-2-1-released-bitcoin-cash-and-zcash-integrated/) we released [Bitcoin Cash](https://www.bitcoincash.org/) and [ZCash (Beta)](https://z.cash) support for OpenBazaar. We are proud that we’ve been able to bring OpenBazaar to a broader community of crypto enthusiasts and are continuing that work today. 
+
+#### Wonder What's Next?
+
+In 2018 the breadth of crypto projects has increased exponentially through ERC-20 token issuances, completely new blockchains, layer two networks and more. We understand that there is a growing number of people interested in using all kinds of cryptocurrencies—not just Bitcoin—for e-commerce, coin trading, rewarding others for content creation and other use cases. OpenBazaar was always designed as a project for _the world_, not just for Bitcoin, and we’re hard at work re-engineering it to support new coins and communities. 
```

Comment:
> cryptocurrencies—not just Bitcoin—for

AP style is to put a space on either side of the em dash. It's correct without it though, so up to you. I think it's easier to read with the space.